examples/datepicker-plots.py

```diff
@@ -0,0 +1,97 @@
+import pandas as pd
+from dash import Dash, dcc, html, Input, Output, State
+import plotly.express as px
+import dash_bootstrap_components as dbc
+import time
+
+df = px.data.stocks()
+df['date'] = pd.to_datetime(df['date'])
+app = Dash(__name__, external_stylesheets=[dbc.themes.BOOTSTRAP])
+
+app.layout = html.Div([
+    html.Label('Plot Type', style={'margin-bottom': '5px', 'font-weight': 'bold'}),
+    dcc.Dropdown(
+        id='datepicker-plots-x-dropdown',
+        options=[
+            {'label': 'Select a plot', 'value': ''},
```

Comment:
It's better to remove the "select a plot" option and include [clearable=False](https://dash.plotly.com/dash-core-components/dropdown#dropdown-clear)